crates/telemetry-subscribers/src/lib.rs

```diff
@@ -273,6 +273,10 @@ impl TelemetryConfig {
             self.tokio_console = true;
         }
 
+        if let Ok(log_level) = env::var("RUST_LOG") {
```

Comment:
It's an interesting find that `RUST_LOG` is not being read here.  Couple of notes though:

- `RUST_LOG` may not just be a log level, it could also contain [per-module levels/filters](https://docs.rs/env_logger/0.10.0/env_logger/#enabling-logging) so we should check whether this does the right thing given that kind of input. 
  - (Note, that I believe we use `tracing-rs`, instead of `env_logger`, but I couldn't find its docs on how it interprets `RUST_LOG` and AFAICT, it is compatible).
- From what I could tell, logging was behaving badly even when `RUST_LOG` was not set at all, which this change should not affect.  Were you able to reproduce that behaviour, and if so, do these changes fix it?

Reply:
That's what happens when I code late at night (my brain stops functioning). I removed this because indeed was not needed. 

> From what I could tell, logging was behaving badly even when RUST_LOG was not set at all, which this change should not affect. Were you able to reproduce that behaviour, and if so, do these changes fix it?
That's a good point. I built the binaries without this change and gave it a try. The thing is that if one sets `RUST_LOG`, all commands will follow that log level. For example, sui start / genesis/ will all use the RUST_LOG, but so is keytool, move, client etc. It looks like in some commands we might want to use `info!` (e.g., in keytool) for easier visual debugging. Therefore, I think probably setting the log level for some commands to error is a good thing, and then the user can overwrite by setting the variable. 

For some of the commands (sui start, network, genesis, etc), we should keep info level.